utils/flist/techmap_vhdl.flist

```diff
@@ -1,39 +0,0 @@
-techmap/gf12/memory_gf12.vhd
-techmap/gf12/clkgen_gf12.vhd
-techmap/gf12/pads_gf12.vhd
-techmap/virtex7/mul_unisim.vhd
-techmap/virtex7/ddr_unisim.vhd
-techmap/virtex7/memory_unisim.vhd
-techmap/virtex7/sysmon_unisim.vhd
-techmap/virtex7/buffer_unisim.vhd
-techmap/virtex7/spictrl_unisim.vhd
-techmap/virtex7/pads_unisim.vhd
-techmap/virtex7/tap_unisim.vhd
-techmap/virtex7/clkgen_unisim.vhd
-techmap/virtex7/sgmii_vcu118.vhd
-techmap/virtex7/sgmii_vcu118_clock_reset.vhd
-techmap/virtex7/sgmii_vc707.vhd
-techmap/virtexu/mul_unisim.vhd
-techmap/virtexu/ddr_unisim.vhd
-techmap/virtexu/memory_unisim.vhd
-techmap/virtexu/sysmon_unisim.vhd
-techmap/virtexu/buffer_unisim.vhd
-techmap/virtexu/spictrl_unisim.vhd
-techmap/virtexu/pads_unisim.vhd
-techmap/virtexu/tap_unisim.vhd
-techmap/virtexu/clkgen_unisim.vhd
-techmap/virtexu/sgmii_vcu118.vhd
-techmap/virtexu/sgmii_vcu118_clock_reset.vhd
-techmap/virtexu/sgmii_vc707.vhd
-techmap/virtexup/mul_unisim.vhd
-techmap/virtexup/ddr_unisim.vhd
-techmap/virtexup/memory_unisim.vhd
-techmap/virtexup/sysmon_unisim.vhd
-techmap/virtexup/buffer_unisim.vhd
-techmap/virtexup/spictrl_unisim.vhd
-techmap/virtexup/pads_unisim.vhd
-techmap/virtexup/tap_unisim.vhd
-techmap/virtexup/clkgen_unisim.vhd
-techmap/virtexup/sgmii_vcu118.vhd
-techmap/virtexup/sgmii_vcu118_clock_reset.vhd
-techmap/virtexup/sgmii_vc707.vhd
```

Comment:
why are you deleting FPGA files?

Reply:
I am not deleting them. techmap_v*.flist is automatically generated. techmap_vhdl.flist calls pads and clock generator, something I still haven't worked on. Perhaps this file won't be required to be generated as techmap_vlog.flist does.